lib/gollum/templates/wiki_content.mustache

```diff
@@ -40,6 +40,7 @@
 
 	</div>
 
+	{{^preview}}
```

Comment:
Cool! I should have asked before: will this result in a lack of whitespace at the bottom of the preview page? If so we should make the *contents* of the footer div, not the div itself, conditional on `@preview`. Sorry for the lateness

Reply:
Good point. Fixed it in latest commit.